### PR TITLE
selects currency exchange

### PR DESCRIPTION
**Pull request by jonnyalexbh**
## Summary
- load selects currency exchange

## Trello Card
N/A

## Screenshot
Screen Shot 2019-12-12 at 8 51 42 AM

### CODE REVIEW

File: plink-frontend/src/app/screens/currency-list/currency-list.component.ts [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
   ngOnInit() {
-    this.getPrices();
   }
 
-  getPrices() {
-    this.currencyService.getPrice().subscribe(({ prices }: any) => {
-      this.prices = prices;
-    })
+  ngOnChanges() {
+    console.log('list component', this.prices);
```
Comment (betoobernal) [inline plink-frontend/src/app/screens/currency-list/currency-list.component.ts:21]: remove this log

File: plink-frontend/src/app/screens/exchange-currency/exchange-currency.component.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<div class="exchange">
```
Comment (betoobernal) [inline plink-frontend/src/app/screens/exchange-currency/exchange-currency.component.html:1]: I would believe that this tag is needless

File: plink-frontend/src/app/screens/exchange-currency/exchange-currency.component.scss [hunk 3]
```diff
@@ -0,0 +1,107 @@
+
+
+
+@import '../../../scss/variables/colors';
```
Comment (betoobernal) [inline plink-frontend/src/app/screens/exchange-currency/exchange-currency.component.scss:4]: remove this.

Comment (jonnyalexbh) [reply plink-frontend/src/app/screens/exchange-currency/exchange-currency.component.scss:4]: it's ok in the previous one PR

File: plink-frontend/src/app/screens/currency-list/currency-list.component.ts (hunk 1 above)
Comment (jonnyalexbh) [reply plink-frontend/src/app/screens/currency-list/currency-list.component.ts:21]: ready :)

**Review by betoobernal — APPROVED** [review]

**Pull request merged**